### DISCUSSION TITLE
Can the ROG Ally be equipped with two USB-C ports like the Ally X?

### DISCUSSION THREAD

**Discussion by gaowanliang** [osy/XG_Mobile_Station#19] [General]
To be honest, I think most people aren't very interested in using an eGPU with handheld devices, including myself. However, if it were possible to reverse-engineer the XG Mobile interface to provide USB-C functionality, I believe many people would be quite interested. Unfortunately, I have no knowledge of hardware. Is this feasible?

**Comment by osy** [comment]
Yes it's feasible but I have no interest in that.
